cms/djangoapps/contentstore/rest_api/v1/serializers/course_waffle_flags.py

```diff
@@ -156,7 +156,7 @@ def get_enable_course_optimizer(self, obj):
 
     def get_use_react_markdown_editor(self, obj):
         """
-        Method to get the use_react_markdown_editor waffle flag
+        Method to get the use_react_markdown_editor switch
```

Comment:
```suggestion
        Determine whether the new markdown editor should be enabled in the react problem editor
```
Suggested rewording, which avoid referencing the outdated markdown editor switch